weave-js/src/components/PagePanelComponents/Home/Browse2/callTreeHooks.ts

```diff
@@ -31,13 +37,28 @@ export const useRuns = (
   filters: CallFilter,
   opts?: {skip?: boolean}
 ): {loading: boolean; result: Span[]} => {
+  const rowsNode = callsTableNode(streamId);
+
+  // Added as a short term patch before better backend data model. We have to
```

Comment:
This is a very sad reality -> we need the types to be correct when sending over the request. This will cost us an extra second or so to load calls, but is needed to fix non parquet compacted projects